dask_sql/physical/rel/custom/create_experiment.py

```diff
@@ -147,6 +156,19 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         y = training_df[target_column]
 
         if model_class and experiment_class:
+            if type(training_df) == dd.core.DataFrame:
+                if model_class in cpu_classes:
+                    model_class = cpu_classes[model_class]
+                if experiment_class in cpu_classes:
+                    experiment_class = cpu_classes[experiment_class]
```

Comment:
Think this (and other similar patterns) can be consolidated to

```suggestion
                model_class = cpu_classes.get(model_class, model_class)
                experiment_class = cpu_classes.get(experiment_class, experiment_class)
```